components/splashpage/Splashpage.module.css

```
@@ -0,0 +1,9 @@
+.gridContainer {
+  height: 100vh;
+  display: grid;
+  justify-content: center;
+}
+
+img {
```

Make sure to not affect other images
```suggestion
.gridContainer img
```